test/e2e/src/core/e2e_tests.sh

```diff
@@ -131,9 +131,6 @@ createArcCIExtension() {
     if [ ! -z "$CI_ARC_RELEASE_TRAIN" ]; then
        basicparameters="$basicparameters  --release-train $CI_ARC_RELEASE_TRAIN"
     fi
-    if [ ! -z "$CI_ARC_VERSION" ]; then
```

Comment:
Lets keep the version as is and instead set --auto-upgrade-minor-version false

Reply:
addressed